### ISSUE TITLE
Add default action on whole row

### ISSUE THREAD

**Issue by chemix** [ublaboo/datagrid#89]
Hi, 

thanks for great grid component. I miss just a small think ;-)

Because I use a xxxGridFactory, I don't use in grid definition $grid->addColumnLink(...'edit') for quick primary action. 

``` php
    /**
     * @return DataGrid
     */
    public function create()
    {
        $grid = new DataGrid();

        $projectRepository = $this->em->getRepository(Project::class);
        $grid->setDataSource($projectRepository->createQueryBuilder('project'));

        $grid->addColumnText('name', 'Name');

        /**
         * Filters
         */
        $grid->addFilterText('name', 'Search', ['project.name']);

        return $grid;
    }
```

And in presenter I use 

```php
    protected function createComponentProjectGrid()
    {
        $grid = $this->projectGridFactory->create();

        /** Action */
        $grid->addAction('edit', 'Edit', 'edit')
            ->class('btn btn-default btn-xs')
            ->setIcon('pencil');

       /** Row action */
       // not implemented yet ;-)

        return $grid;
    }
```

What I miss is add some whole line default action

```php
$grid->addRowAction('detail', 'detail');
```

What do you think?

**Comment by paveljanda** [comment]
"whole line default action":

Where should it be rendered? Along all columns? I just don't understand how that is different from `ColumnLink`?
You mean something that would enclose all text/number/... columns in link tag?

**Comment by chemix** [comment]
Ou, give me a second. Maybe I miss some in documentation and grid has this feature ;-)

**Comment by paveljanda** [comment]
@chemix Did you find appropriate functionality, or is there something missing? :)

**Comment by foxycode** [comment]
@paveljanda @chemix wanted to point to two things:
1. When you want to have grid factory reusable, it should be decoupled from presenter. So it is bad idea to use `addColumnLink` in factory, because it's linked to presenter action.
2. There should be simple way how to make whole row as link do edit/detail in presenter. Calling `addAction` adds only button, but you can't make all columns clickable without naming them all and call  `addColumnLink` on them. So method as `addRowAction` would be helpfull.

In short, we need a way, hot to make all columns of row clickable by calling one method :smile:

**Comment by paveljanda** [comment]
I am sorry but that i won't implement. It can lead to disorientation in application code. It's just that "Some big method that will change all columns into type Link" sounds strange to me.

If I were you I would probably do some magic in GridFactory or something like that.
